.circleci/config.yml

```diff
@@ -148,7 +148,7 @@ jobs:
           at: .
       - run:
           name: Generate documentation
-          command: pipenv run ./dev-tools/generate_documentation.sh
+          command: pipenv run ./dev-tools/generate_documentation.sh --clean
```

Comment:
The documentation bulid in circleci is always clean, because the directory in which the intermediate sphinx files are stored is not passed to later builds. So no need to pass this parameter explicitly.